models/DataObject/ClassDefinition/Data/ReverseObjectRelation.php

```diff
@@ -200,4 +196,14 @@ public function getFieldType(): string
     {
         return 'reverseObjectRelation';
     }
+    /**
+     * @inheritdoc
+     */
+    public function getClasses(): array
+    {
+        if($this->ownerClassId) {
+            return Model\Element\Service::fixAllowedTypes([$this->ownerClassId], 'classes');
```

Comment:
```suggestion
            return Model\Element\Service::fixAllowedTypes([$this->ownerClassName], 'classes');
```
Class IDs can be alphanumeric. Otherwise it will try to load the class with name by passing id https://github.com/pimcore/pimcore/blob/e22a8209fde57c953a5caba13292d60ae3b06979/models/DataObject/ClassDefinition/Data/ManyToManyObjectRelation.php#L438